### PR TITLE
improve camel to snake

### PR DESCRIPTION
**Pull request by wakamex**

### CODE REVIEW

**Review by wakamex — COMMENTED** [review]
lgtm

File: elfpy/utils/apeworx_integrations.py [hunk 1]
```diff
@@ -600,7 +600,7 @@ def _snake_to_camel(_snake):
 
 def _camel_to_snake(_camel):
```
Comment (sentilesdal) [inline elfpy/utils/apeworx_integrations.py:601]: typo, shoud be `_camel_to_snek`

**Review by sentilesdal — COMMENTED** [review]
🐍

**Comment by wakamex** [comment]
implemented in #538

**Pull request closed without merging**